bigframes/dataframe.py

```diff
@@ -3674,7 +3674,11 @@ def apply(self, func, *, axis=0, args: typing.Tuple = (), **kwargs):
             return result_series
 
         # Per-column apply
-        results = {name: func(col, *args, **kwargs) for name, col in self.items()}
+        if hasattr(func, "bigframes_remote_function"):
```

Comment:
@TrevorBergeron could you please review if this element-wise proxy for series-wise application doesn't have any blind sight?

Reply:
Isn't `dataframe.apply` for column or series-wise application? Why do we need to support applying as an element-wise operator? isn't this what `dataframe.map` is for?

Reply:
For scalar ops the end result is the same. I think we don't lose anything by being lax for user convenience. Note that this API has other non-pandas-like extended behavior https://github.com/googleapis/python-bigquery-dataframes/blob/e0a82888cd34fa2404ac68229dc38496cb22c67b/bigframes/dataframe.py#L3933